brickstrap-unshare-harder.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
I'm not sure how I feel about this. For starters, this is targeted at Debian because Debian doesn't enable `unprivileged_userns_clone` by default. But, Debian also does not use `sudo` by default, so this still won't work by default.

Also, couldn't this just be included in the main brickstrap.sh script?

I would be happy just adding a check for this to the main script and provide instructions on how to fix it manually, similar to checking that the kernel image is readable that is already done.